app/models/distribution.rb

```diff
@@ -42,11 +42,16 @@ def schedule
       days << :saturday if saturday
       days << :sunday if sunday
       schedule.rrule(IceCube::Rule.weekly.day(days))
+      if special_event == "1"
```

Comment:
le `special_event` --> "1" c'est le ramadan c'est ça? Si oui, ce serait intéressant de gérer ça avec des enums potentiellement. Je te laisse regarder si tu trouves ça cool : https://robots.thoughtbot.com/whats-new-in-edge-rails-active-record-enum